sdk/keyvault/Azure.ResourceManager.KeyVault/api/Azure.ResourceManager.KeyVault.netstandard2.0.cs

```diff
@@ -436,10 +436,27 @@ public partial class ServiceSpecification
     }
     public partial class Sku
     {
-        public Sku(Azure.ResourceManager.KeyVault.Models.SkuName name) { }
-        public string Family { get { throw null; } set { } }
+        public Sku(Azure.ResourceManager.KeyVault.Models.SkuFamily family, Azure.ResourceManager.KeyVault.Models.SkuName name) { }
+        public Azure.ResourceManager.KeyVault.Models.SkuFamily Family { get { throw null; } set { } }
         public Azure.ResourceManager.KeyVault.Models.SkuName Name { get { throw null; } set { } }
     }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct SkuFamily : System.IEquatable<Azure.ResourceManager.KeyVault.Models.SkuFamily>
+    {
+        private readonly object _dummy;
```

Comment:
This is generated using the reference assembly generator that's owned by BCL team. It adds _dummy fields because of the reasons outlined in https://blog.paranoidcoding.com/2016/02/15/are-private-members-api-surface.html.